pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -48,4 +49,5 @@ func (s *DeployFlags) Set(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVar(&s.Logs, "logs", true, fmt.Sprintf("Show logs from Pods annotated as '%s'", deployLogsAnnKey))
 	cmd.Flags().BoolVar(&s.LogsAll, "logs-all", false, "Show logs from all Pods")
+	cmd.Flags().StringVar(&s.AppMetadataFile, "appMetadataFile", "", "write app metatadata to file")
```

Comment:
`app-metadata-file`?

Reply:
yeah good suggestion! i'll change it now